javascript/funWithFunctions.md

```diff
@@ -0,0 +1,24 @@
+## Functional Expressions in D&D ##
```

Comment:
I'm not against having a more story-like TIL. Do you think this would fit better as a blog post? Thoughts @mjankowski @croaky ?